news/137.bugfix

```diff
@@ -0,0 +1 @@
+Avoid accidental acquisition in block_types indexer. @davisagli
```

Comment:
I noticed inconsistent use of inline literals in `CHANGES.rst`. Some use Markdown's single backticks, others use reStructuredText's double backticks, and still more use none.

```suggestion
Avoid accidental acquisition in ``block_types`` indexer. @davisagli
```